### ISSUE TITLE
Scrollbar causes page content to shift on log in/sign up page

### ISSUE THREAD

**Issue by 8thDay** [chingu-voyage4/Geckos-Team-26#18]
Regarding the user form (log in/sign up page):
The page content is moving about as I switch between the "log in" and "sign up" forms. The "sign up" page is longer and is creating a scrollbar, where the "log in" page has none.

My default fix for this would be to force a scrollbar on both pages. Is there a better way?

**Comment by mairamartinsk** [comment]
I'll take a look at it!

**Comment by mairamartinsk** [comment]
@8thDay The scrollbar is due to the signup form having more content and making the page longer than the viewport height. The login form has less content and, at least on my screen, the footer is not even touching the bottom of the viewport.

I'm not sure how to fix this, since the two forms are actually one page only, or rather, one component. I suggest we leave this issue open a little longer.

**Comment by 8thDay** [comment]
I'm happy to leave this open for now. 👍 

The best I can come up with atm, is to use something like this:
```
html {
  min-height: 100vh;
  margin-bottom: 1px;     // perhaps border-bottom:1px; depending which box model we are using
}
```

It does stop the horizontal movement, but it doesn't feel like a very elegant solution.

I don't suppose that semantic.ui has any helper classes that would deal with this?

**Comment by mairamartinsk** [comment]
Fixed with PR #47 !

Semantic UI was of no help. Your solution was it, only 100vh was too much. I made a global class with min-height: 70vh and applied to all components that needed it (userform, notfound, dashboard).
